packages/make-hardener/package.json

```
@@ -25,8 +25,14 @@
     "test": "test"
   },
   "type": "module",
-  "main": "./src/main.js",
-  "browser": "dist/make-hardener.umd.js",
+  "main": "./dist/make-hardener.cjs",
+  "module": "./src/main.js",
+  "browser": "./dist/make-hardener.umd.js",
```

Let's support unpkg.com as we do for SES.

```suggestion
  "browser": "./dist/make-hardener.umd.js",
  "unpkg": "./dist/make-hardener.umd.js",
```